ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/remoteTesting/FinalizedDataModelIsMarkedAsFinalizedInXMLSpec.groovy

```diff
@@ -42,5 +47,28 @@ class FinalizedDataModelIsMarkedAsFinalizedInXMLSpec extends GebSpec {
 
         then:
         dataModelPage.isExportVisible()
+
+        when:
+        List<String> files = []
+        String downloadPath = System.getProperty("downloadFilepath")
+
+        File file = new File(downloadPath)
+        File cancelFile = new File(downloadPath + "Cancer_Model.mc.xml")
+
+        if (cancelFile.exists()) {
+            cancelFile.delete()
+        }
+
+        dataModelPage.export()
+        dataModelPage.exportXml()
+        Thread.sleep(2000)
+
+        file.eachFile(FileType.FILES) {
+            files.add(it.name)
+        }
+
+        then:
+        files.contains("Cancer_Model.mc.xml")
+        cancelFile.text.contains(dataModelName)
```

Comment:
@Vendetta30 this should check that the status is Finalized in the XML. See attachment

<img width="744" alt="napkin 7 02-04-18 10 01 24 am" src="https://user-images.githubusercontent.com/864788/38188568-cde5200a-365c-11e8-867b-9c45ea23c91c.png">